.buildkite/pipeline.verify.yml

```diff
@@ -229,6 +229,21 @@ steps:
         artifact_paths:
           - "test_artifacts/**/*"
 
+      - label: ":typescript: Typescript Integration Tests"
+        command:
+          - make test-integration-typescript
+        plugins:
+          - grapl-security/vault-login#v0.1.3
+          - grapl-security/vault-env#v0.2.0:
+              secrets:
+                - grapl/TOOLCHAIN_AUTH_TOKEN
+        env:
+          GRAPL_LOG_LEVEL: "DEBUG"
```

Comment:
Removed

Reply:
I was referring to the `GRAPL_LOG_LEVEL` environment variable, not the `TOOLCHAIN_AUTH_TOKEN` secret. As currently written, this is an invalid Buildkite step definition because of the incomplete `vault-env` plugin specification.

Reply:
Removed `LOG_LEVEL`, ADDED BACK `TOOLCHAIN_AUTH_TOKEN` secret